src/templates/Footer/Footer.tsx

```diff
@@ -0,0 +1,90 @@
+import { createTheme, ThemeProvider } from '@mui/material/styles';
+import { Grid, Link, Typography, useMediaQuery } from '@mui/material';
+import { IconsGroup } from '../../components/IconsGroup/IconsGroup';
+
+declare module '@mui/material/styles' {
+  interface BreakpointOverrides {
+    xs: false;
+    sm: false;
+    md: false;
+    lg: false;
+    xl: false;
+    base: true;
+    mobile: true;
+    tablet: true;
+    desktop: true;
+  }
+}
+const defaultTheme = createTheme({
+  breakpoints: {
+    values: {
+      base: 0,
+      desktop: 1024,
+      mobile: 360,
+      tablet: 768,
+    },
+  },
+});
```

Comment:
This should be in a separated file + you should pass our customTheme into root theme provider. As this breakpoint will be used across the whole application:
https://mui.com/material-ui/customization/theming/#custom-variables

Reply:
Changed in 2b03974